tests/gptq/test_quantization.py

```diff
@@ -0,0 +1,174 @@
+# coding=utf-8
+# Copyright 2023 The HuggingFace Team. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import tempfile
+import unittest
+
+import torch
+from parameterized import parameterized
+from transformers import AutoModelForCausalLM, AutoTokenizer
+
+from optimum.gptq import GPTQQuantizer, load_quantized_model
+from optimum.gptq.data import get_dataset
+from optimum.utils.testing_utils import require_accelerate, require_autogptq, require_torch_gpu
+
+
+@require_autogptq
+@require_torch_gpu
+class GTPQTest(unittest.TestCase):
```

Comment:
can you make these tests slow? We probably don't want to run them in the PR/main CI given that they use large models. You would need to add a yml file in .github/workflows as well for gptq.

Reply:
Done for the slow tests. I will update the workflows later